package.json

```diff
@@ -17,8 +17,16 @@
     "build:umd": "cross-env BABEL_ENV=commonjs webpack",
     "build:umd:min": "cross-env BABEL_ENV=commonjs NODE_ENV=production webpack",
     "build:watch": "npm run build:commonjs -- -watch",
+    "precommit": "lint-staged",
     "prepublish": "npm run clean && npm run build"
   },
+  "lint-staged": {
```

Comment:
do you have an alternate recommendation for how to implement prettier? I would like to ensure all code PRed is prettier'ified, wether that is via CI or lint-staged or other.